sdk/devcenter/azure-developer-devcenter/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (Unreleased)
+This Azure DevCenter library release uses the 2023-04-01 GA API.
```

Comment:
nit, avoid use acronym (GA). Spell the full word.

Personally opinion. I'd expect a better summary for the 1st GA release.